### PR TITLE
fix(discv2): hatch text color

### PR DESCRIPTION
**Pull request by doralchan**
**Before:** 
Screen Shot 2020-01-02 at 11 23 41 AM
**After:**
Screen Shot 2020-01-02 at 3 21 05 PM

### CODE REVIEW

**Review by leedongwei — DISMISSED** [review]
This hatch is so fetch.

**Review by leedongwei — COMMENTED** [review]
Didn't see it was WIP 😅

**Comment by dashed** [comment]
@leedongwei no worries. I un-did your review 🙃

**Comment by leedongwei** [comment]
So weird that I cannot undo my own review.

**Comment by doralchan** [comment]
Not sure why this isn't working. It should?

**Comment by doralchan** [comment]
I believe this fixed it @dashed

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx [hunk 1]
```diff
@@ -679,20 +679,13 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
     dividerHandlerChildrenProps: DividerHandlerManager.DividerHandlerManagerChildrenProps
   ) => {
     const {span, spanBarColour, spanBarHatch, spanNumber} = this.props;
-
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx]: my whitespace 😭

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**